src/StaticFAQ/FAQExtraInfo/BaggageInfo/__test__/BaggageDescription.test.js

```diff
@@ -8,15 +8,18 @@ import { BaggageDescription } from '../BaggageDescription';
 const mockRefType: any = null;
 const baggage = {
   $refType: mockRefType,
-  height: 40,
-  width: 30,
-  length: 15,
-  weight: 10,
+  bag: {
+    height: 40,
+    width: 30,
+    length: 15,
+    weight: 10,
+    category: 'CHECKED',
+    note: 'asd',
+  },
+  quantity: 0,
```

Comment:
will the GraphQL return bag if quantity is 0?

Reply:
Good question. GraphQL currently relies on the API to handle this edge case. I believe the API has to handle this, but to be safe maybe we could just handle it on GraphQL too.